### PR TITLE
Avoid flakiness on external/wpt/css/css-ui-3/outline-004.html

### PR DESCRIPTION
**Pull request by jeffcarp**
Avoid flakiness on external/wpt/css/css-ui-3/outline-004.html

This patch uses "ahem" instead of "Ahem" to avoid flakiness
on outline-004.html test. And removes it from TestExpectations.

BUG=719299

Change-Id: Ic14b7a7d8f0db0c0c9a5501dd91032cd2758549b
Reviewed-on: https://chromium-review.googlesource.com/506009
Reviewed-by: Quinten Yearsley <qyearsley@chromium.org>
Cr-Commit-Position: refs/heads/master@{#471924}
WPT-Export-Revision: 646352396b3879f156c342846a52230fffaa3e21

### CODE REVIEW

**Comment by jeffcarp** [comment]
This PR corresponds to a currently in-flight CL: https://chromium-review.googlesource.com/c/506009/

Please don't merge it manually - once the upstream CL lands and if Travis is green, the exporter will auto-merge it.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232615993)

# Firefox (nightly) #
Testing web-platform-tests at revision 62b47b161d18f1ecdf6e4dacd416b52f27e82270
Using browser at version BuildID 20170515100238; SourceStamp e66dedabe582ba7b394aee4f89ed70fe389b3c46
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5928/css/css-ui-3/outline-004.html">/css/css-ui-3/outline-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232615994)

# Sauce (safari) #
Testing web-platform-tests at revision 62b47b161d18f1ecdf6e4dacd416b52f27e82270
Using browser at version 10.0
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5928/css/css-ui-3/outline-004.html">/css/css-ui-3/outline-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232615995)

# Sauce (MicrosoftEdge) #
Testing web-platform-tests at revision 62b47b161d18f1ecdf6e4dacd416b52f27e82270
Using browser at version 14.14393
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>1 test ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5928/css/css-ui-3/outline-004.html">/css/css-ui-3/outline-004.html</a></summary>


| Subtest | Results | Messages |
|---------|---------|----------|
|         | FAIL    |          |

</details>
</details>

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/232616003)

# Chrome (unstable) #
Testing web-platform-tests at revision 62b47b161d18f1ecdf6e4dacd416b52f27e82270

**Review by frivoal — APPROVED** [review]
The change looks good to me, so approving.

That said, the comment also says “it adds a new test font-family-case-insensitive.htmlusing "ahem"”, but that test isn't there.

**Comment by mrego** [comment]
Thanks for the review.

I've changed the commit message, as it was only for Blink.
We already have the `font-family` casing stuff covered in `css/css-fonts-3/font-family-name-*.html` tests.

**Comment by foolip** [comment]
Note that commit message from Chromium can sometimes be somewhat nonsensical and refer to things that aren't included in the upstreamed commit. We've kind of just accepted this, as this happens for Gecko as well.

**Pull request merged**
